app/v1/models/response/retrieveEmploymentExpenses/RetrieveEmploymentExpensesResponse.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.response.retrieveEmploymentExpenses
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, Json, OWrites, Reads}
+import v1.models.des.DesSource
+import v1.models.domain.MtdSource
+
+case class RetrieveEmploymentExpensesResponse(submittedOn: Option[String],
+                                              totalExpenses: Option[BigDecimal],
+                                              source: Option[MtdSource],
+                                              dateIgnored: Option[String],
+                                              expenses: Option[Expenses])
+
+object RetrieveEmploymentExpensesResponse {
+
+  implicit val reads: Reads[RetrieveEmploymentExpensesResponse] = (
+    (JsPath \ "submittedOn").readNullable[String] and
+      (JsPath \ "totalExpenses").readNullable[BigDecimal] and
+      (JsPath \ "source").readNullable[DesSource].map(_.map {
+        case DesSource.`HMRC HELD` => DesSource.`HMRC HELD`.toMtd
```

Comment:
This can probably be shortened to something like 
```
(JsPath \ "source").readNullable[DesSource].map(_.map(_.toMtd))
```
I think the one-to-one mapping is probably redundant now you're reading it as a DesSource instead of a String